Assets/Scripts/UI/CombatHUD/CombatTutorial.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CombatTutorial : MonoBehaviour
+{
+	private bool tutorialOn;
+	private void Awake()
+	{
+		tutorialOn = true;
+	}
+
+	private float timer = 0.0f;
+	public float secondsToDisplay;
+	private void Update()
+	{
+		timer += Time.deltaTime;
+		if ((timer > secondsToDisplay) && tutorialOn)
+		{
+			toggleTutorial();
+		}
+	}
+
+	public void toggleTutorial()
+	{
+		tutorialOn = !tutorialOn;
+		gameObject.SetActive(tutorialOn);
+	}
```

Comment:
Same as above, should be PascalCase ( I think? https://github.com/Liam-Harrison/Celeritas/wiki/Programming-Conventions )

Reply:
OKAY SO FUNNY STORY, I originally had it as ToggleTutorial but thought the better of it and renamed it.  Like a dumbass.  Should've looked at the docs, self!

Anyway the reason the button wasn't working is because Unity doesn't update method names, so it was still trying to call ToggleTutorial when such a method didn't exist.  Changing it all back now, though!